src/bitcoin-cli.cpp

```diff
@@ -151,10 +151,15 @@ static int AppInitRPC(int argc, char* argv[])
             strUsage += FormatParagraph(LicenseInfo());
         } else {
             strUsage += "\n"
-                "Usage:  bitcoin-cli [options] <command> [params]  Send command to " PACKAGE_NAME "\n"
-                "or:     bitcoin-cli [options] -named <command> [name=value]...  Send command to " PACKAGE_NAME " (with named arguments)\n"
-                "or:     bitcoin-cli [options] help                List commands\n"
-                "or:     bitcoin-cli [options] help <command>      Get help for a command\n";
+                "The bitcoin-cli utility provides a command line interface to interact with a " PACKAGE_NAME " RPC server.\n"
+                "\nIt can be used to query network information, manage wallets, create or broadcast transactions, and control the " PACKAGE_NAME " server.\n"
+                "\nUse the \"help\" command to list all commands. Use \"help <command>\" to show help for that command.\n"
+                "Using the -named option allows specifying params using key=value style, and therefore doesn't require passing unused params.\n"
```

Comment:
```suggestion
                "The -named option allows specifying parameters using key=value instead of positional style.\n"
```

Reply:
Re-worded in 4aff4ea92c96f4f66e0159329ad8d01d4cf7bce7